rasa/engine/recipes/default_recipe.py

```diff
@@ -477,7 +477,7 @@ def _add_core_train_nodes(
                 config_key: cli_parameters[param]
                 for param, config_key in {
```

Comment:
Can this just be a list now as the param and config_key are always the same?

Reply:
Yep, was unsure if I should change it or not in case we needed to do something later - will do so now